src/tmplits/tmplits-led/module.js

```diff
@@ -70,7 +70,9 @@ export function TmplitLed(context, args) {
     if (context[0]){
         classList = classList.concat(['input-group', 'form-control', 'label-led' ])
         label = `<div class='led-label' style='${labelStyle}' >${finalResult}</div>`
-    } 
+    } else {
+        classList = classList.concat(['tmplit-led-unlabled-container'])
```

Comment:
Echoing @sclaiborne 's earlier note from earlier in this file: 

It would be more efficient to use `push` here instead, so as to not create a new array each time.